stf/build.gradle

```diff
@@ -37,46 +18,26 @@ sourceSets {
     }
 }
 
-manifestDependencies {
-    eclipseGroup = 'eclipse-deps'
-
-    dependencyHandling {
-        def ids = [:]
-
-        return ids
-    }
-}
-
 dependencies {
     compile junitVersion
     compile project(':saros.core')
     compile project(':saros.eclipse')
 
     compile project(path: ':saros.eclipse', configuration: 'testing')
 
-    releaseDep fileTree(dir: 'libs', include: '*.jar', exclude: '*source*.jar')
-    compile fileTree(dir: eclipseHome + '/plugins').matching {
+    // Required for the whiteboard stf component
+    compile files('libs/org.eclipse.draw2d_3.6.2.v20110128-0100.jar', 'libs/org.eclipse.gef_3.6.2.v20110110-2020.jar')
```

Comment:
Why are these libraries loaded explicitly? When the whiteboard is build/tested, shouldn't it already provide these libraries?

Reply:
The stf contains methods in order to test the wb. Therefore the stf uses gef stuff.
I don't like to use static libs in our repo, but I don't want to invest time before we
know what we do with the whiteboard.